make/migrations/postgresql/0035_2_1_0_schema.up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE blob ADD COLUMN IF NOT EXISTS update_time timestamp default CURRENT_TIMESTAMP;
+ALTER TABLE blob ADD COLUMN IF NOT EXISTS status varchar(255);
+ALTER TABLE blob ADD COLUMN IF NOT EXISTS version BIGINT default 0;
```

Comment:
Please add index for the status and version columns.

Reply:
added